front/src/contexts/UploadArtworkContext.tsx

```diff
@@ -106,6 +120,10 @@ export const UploadArtworkProvider: React.FC = ({ children }) => {
           ? "SHARE_TO_SLACK_WITH_IMAGE"
           : "SHARE_TO_SLACK"
         : "NONE";
+      const twitterShareOption = {
+        share: notifyTwitter,
+        userName: twitterUserName,
```

Comment:
[MUST]
`username` は全部小文字ですね。
```suggestion
        username: twitterUserName,
```

https://github.com/kmc-jp/GodUploader-graphql/blob/f9ae19f760590d7df9b7c00a18a4a25d09c1f12d/front/schema.graphql#L390-L391